### PR TITLE
[EMCAL-724, EMCAL-730] QC fixes for global processing

### PR DESCRIPTION
**Pull request by mfasDa**
- [EMCAL-730] Align hostname to global processing
- [EMCAL-724] Update QCDB relocator in stand-alone configs

### CODE REVIEW

**Comment by mfasDa** [comment]
Needed as qc-workflow.sh uses localhost as hostname, so using localMachines "epn" will lead to no local task found.

File: DATA/testing/detectors/EMC/qc/emcQCTasks_singlenode.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	    "config": {
 		"database": {
 		    "implementation": "CCDB",
-		    "host": "alio2-cr1-hv-qcdb1-gpn.cern.ch:8083",
+		    "host": "aalio2-cr1-hv-qcdb1.cern.ch:8083",
```
Comment (davidrohr) [inline DATA/testing/detectors/EMC/qc/emcQCTasks_singlenode.json:6]: Is this correct with 2 `a`?

**Review by davidrohr — APPROVED** [review]
I assume this addresses the issue you reported yesterday?
(I was currently looking into it, so I can stop now?)

File: DATA/testing/detectors/EMC/qc/emcQCTasks_singlenode.json (hunk 1 above)
Comment (mfasDa) [reply DATA/testing/detectors/EMC/qc/emcQCTasks_singlenode.json:6]: No, typo, sorry

**Comment by mfasDa** [comment]
Typo fixed, @davidrohr thanks a lot for spotting!

**Pull request merged**